src/Chapter18/Listing18.20.AttributeUsageAttributeRestrictingWhereToApplyAnAttribute.cs

```diff
@@ -1,8 +1,13 @@
 namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter18.Listing18_20
 {
+    //will not compile if uncommented
```

Comment:
```suggestion
    // Will not compile if uncommented
```
nit: styling of comment